watcard_office.h

```diff
@@ -25,6 +25,7 @@ _Task WATCardOffice {
 		Printer &prt;
 		void main();
 	public:
+		bool done=false;
```

Comment:
should this be public?

Reply:
yes, otherwise WATCardOffice can't set it to true on destructor